### PR TITLE
Add user, group, and timeout properties to osl_gitlfs resource

### PR DESCRIPTION
**Pull request by daviddjh**
Changes being made for proj-lf:archive

### CODE REVIEW

**Comment by daviddjh** [comment]
[Fix missing images for http://events17.linuxfoundation.org/events/kubecon-and-cloudnativecon-europe](https://trello.com/c/Z8YhtPRU/46-fix-missing-images-for-http-events17linuxfoundationorg-events-kubecon-and-cloudnativecon-europe)

File: resources/osl_gitlfs.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+provides :osl_gitlfs
 
 default_action :sync
 
 provides :osl_gitlfs
 property :destination, String, name_property: true
 property :repository, String
+property :user, [String, Integer]
+property :group, [String, Integer]
+property :timeout, Integer
```
Comment (ramereth) [inline resources/osl_gitlfs.rb:11]: These really should have some default values I think.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by daviddjh** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.5.0.  Have a nice day!

**Pull request merged**